docs/foundation/Icons/Library.stories.tsx

```diff
@@ -39,6 +43,15 @@ const classes = {
     maxWidth: 400,
     paddingBottom: "20px",
   }),
+  callout: css({
+    width: "100%",
+    justifyContent: "center",
+    alignItems: "center",
+    backgroundColor: theme.colors.primary_20,
```

Comment:
Now that we can display the banner only when on `pentahoPlus` theme, can we use warning instead? Users shouldn't be using these icons
```suggestion
    backgroundColor: theme.colors.warning_20,
```